File: patterns/lichtsteuerungssystem/rules-python/lichtsteuerungssystem_zeit.py
from openhab import rule, Registry
from openhab.triggers import GenericCronTrigger


@rule(triggers=[GenericCronTrigger("0 30 6 ? * * *")])
class AdjustLightLevelsBasedOnTime:
    def execute(self, module, input):
        Registry.getItem("PresetDimLevel").postUpdate(20)
        Registry.getItem("UpdateLightLevels").sendCommand("ON")

File: patterns/lichtsteuerungssystem/rules-js/lichtsteuerungssystem-zeit.js
rules.JSRule({
  name: "Adjust light levels based on time",
  triggers: [triggers.GenericCronTrigger("0 30 6 ? * * *")],
  execute: (event) => {
    items.getItem("PresetDimLevel").postUpdate(20);
    items.getItem("UpdateLightLevels").sendCommand("ON");
    console.log("PresetDimLevel -> 20, UpdateLightLevels -> ON");
  }
});
